appveyor.yml

```diff
@@ -1,12 +1,18 @@
 version: '{branch}-{build}'
-build: off
+build: false
+image: Visual Studio 2013
 cache:
   - '%LOCALAPPDATA%\pip\Cache'
 matrix:
   fast_finish: true
 environment:
   global:
+    DEV_BUCKET: watchmaker-dev
+    RELEASE_BUCKET: watchmaker
+    SATS_TOKEN:
```

Comment:
would prefer this to be set in the appveyor service, and removed from appveyor.yml